ui/pages/Dashboard/Dashboard.jsx

```diff
@@ -1,11 +1,5 @@
 import React from 'react'
-import DocumentTitle from 'react-document-title'
 
 import ProjectsTable from './components/ProjectsTable'
 
-export default () => (
-  <div>
-    <DocumentTitle title="seqr: home" />
-    <ProjectsTable />
-  </div>
-)
+export default () => <ProjectsTable />
```

Comment:
The component doesn't add any new attributes. Can we get rid of it and use `ProjectsTable` directly?